pkg/controllers/vdb/podfacts.go

```diff
@@ -157,6 +157,10 @@ type PodFact struct {
 
 	// Is the agent running in this pod?
 	agentRunning bool
+
+	// Check if the pod has the agent keys
+	// and an API Key, in apikeys.dat, used for agent calls
+	hasAgentKeys bool
```

Comment:
Doesn't this mean if the image has the agent keys?
```suggestion
	// Check if the image has agent keys saved in the dbadmin directory.
	imageHasAgentKeys bool
```